product_configurator_mrp_component/models/mrp_bom_line_config.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2022-Today Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class MRPBoMLineConfig(models.Model):
+    _name = "mrp.bom.line.config"
```

Comment:
Description needed to avoid warnings on the server log
```suggestion
    _name = "mrp.bom.line.config"
    _description = "BOM Line Configuration"    
```